algorithms/fd/dqn_agent.py

```diff
@@ -240,6 +138,7 @@ def write_log(self, i: int, avg_loss: dict, score: int = 0):
                 avg_loss[0],
                 avg_loss[1],
                 avg_loss[2],
+                datetime.datetime.now(),
```

Comment:
Why do you log datetime on dqn?

Reply:
That is to know how much time flies. It prints out the current time on log.